src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -482,6 +503,16 @@ def return_response(self):
         else:
             object_key = self._push_content()
             file_name = 'hca-manifest-' + object_key.rsplit('/', )[-1]
+
+        if self.format == 'full':
+            try:
+                file_name = self.storage_service.get(object_key + self.name_file_suffix).decode()
+            except ClientError as e:
+                if e.response['Error']['Code'] == 'NoSuchKey':
+                    logger.warning(f"{object_key}{self.name_file_suffix} doesn't exist. Renaming the filename of the"
```

Comment:
The warning message is crap. Please prefer % over f string in log messages. This leaves file_name unassigned.